api/extension-capabilities/theming.md

```diff
@@ -18,10 +18,10 @@ In Visual Studio Code, there are two types of themes:
 
 ![color-theme](images/theming/color-theme.png)
 
-As you can see in the illustration, Color Theme defines two mappings:
+As you can see in the illustration, Color Theme defines colors for UI components as well as for highlighting in the editor:
 
 - The `colors` mapping that controls colors for UI Components.
-- The `tokenColors` mapping that controls colors for each source code token (your source code is broken into tokens by a [grammar](/api/language-extensions/syntax-highlight-guide)).
+- The `tokenColors`and the `semanticTokenColors` mappings as well as the `semanticHighlighting` setting define the color and styles for the syntax and semantic highlighting in the editor. The [Syntax Highlighting Guide](/api/language-extensions/syntax-highlight-guide) has more information on that topic.
```

Comment:
This is long and a bit confusing. I don't think user can easily digest this. I think we should separate the semantic highlighting guide apart from the TextMate guide, and add one more bullet point here pointing to that.